app/views/guidance-for-farmers.njk

```diff
@@ -279,18 +283,38 @@
         You must make sure the review takes place and you claim funding within 6 months of receiving your agreement number.
       </p>
 
+      <p class="govuk-body">
+        If the review has not taken place and you need more time to complete the review, contact the RPA.
+      </p>
+      <p class="govuk-body">
+        If the review has taken place, and you have a good reason why you won’t be able to make your claim within 6 months - such as illness, contact the RPA as soon as you can.
+      </p>
+
       <h2 id="gethelp" class="govuk-heading-m">Get help with your application or claim</h2>
 
       <p class="govuk-body">
-        If you cannot apply or claim online or have a question about your application or claim, contact the Rural Payments Agency.
+        Contact the RPA if you:
       </p>
-      <p class="govuk-body">Have your agreement number with you when you call or include it in your email if you have it.</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>decide you do not want a review after you’ve accepted the agreement offer</li>
+        <li>cannot apply or claim online or have a question about your application or claim</li>
+      </ul>
+      
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your email or letter.</p>
+      
       <ul class="govuk-list">
         <li>Rural Payments Agency</li>
+        <li>PO Box 352</li>
+        <li>Worksop</li>
+        <li>S80 9FG</li>
+      </ul>
+
+      <ul class="govuk-list">
```

Comment:
this list together with the above seem to make up one logical unit, semantically they seem one section, can we group them somehow?

<img width="514" alt="image" src="https://user-images.githubusercontent.com/118533775/213436359-8b16c63c-b556-4dc2-8a54-e47c89c6220e.png">

Reply:
List items are not always the same across pages.